packages/config/rollup.js

```diff
@@ -96,6 +105,10 @@ const makeConfig = ({
       input,
       plugins: [
         externals({ deps: false }),
+        modify({
+          find: /'__CITATIONS__'/g,
```

Comment:
This is looking for single quotes, but the pattern in the committed files seems to be double quotes. Might want to verify functionality? Switch to double quotes here?